src/index/DeltaTriples.h

```diff
@@ -142,8 +140,11 @@ class DeltaTriples {
     TriplesToHandlesMap triplesDeleted_;
   };
 
-  State<false> state_;
-  State<true> internalState_;
+  TriplesToHandles<false> state_;
+  TriplesToHandles<true> internalState_;
```

Comment:
The variables should not be called "state" anymore. Suggestion:

`triplesToHandlesNormal_`
`triplesToHandlesInternal_`